src/yadism/coefficient_functions/light/nlo/g1.py

```diff
@@ -0,0 +1,27 @@
+import numba as nb
+import numpy as np
+from eko.constants import TR, CF
+
+from . import f2
+
+
+@nb.njit("f8(f8,f8[:])", cache=True)
+def ns_reg(x, args):
+    return (
+        2
+        * CF
+        * (-(1 + x) * np.log(1 - x) - (1 + x**2) * np.log(x) / (1 - x) + 2.0 + x)
+    )
```

Comment:
```suggestion
def ns_reg(z, args):
    return (
        2
        * CF
        * (-(1 + z) * np.log(1 - z) - (1 + z**2) * np.log(z) / (1 - z) + 2.0 + z)
    )
```
Maybe for consistency let's use $z$ everywhere.